docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: ./.config
       args:
-        grafana_version: ${GRAFANA_VERSION:-9.1.2}
```

Comment:
just curious if this change is necessary, seems harmless, more just curious if the e2e tests use this docker file at all

Reply:
the e2e tests that run in ci don't use this docker file, i was using this to test the e2e locally.